tests/cases/examples/hello.slint

```diff
@@ -170,3 +170,24 @@ Hello := Rectangle {
     property <string> arrow_down_commands: "M21.8,311.1l84.2-82.1c15.7-15.2,41-15.2,56.7,0l341.1,304.1l333.7-297.5c15.5-15.2,41-15.2,56.6,0l84.3,82.1c15.6,15.2,15.6,40,0,55.2L531.7,771c-15.7,15.3-41,15.3-56.7,0l-6.9-6.7L21.8,366.3C6.1,351,6.1,326.3,21.8,311.1z";
     property <string> funky_shape_commands: "M 100 300 Q 150 50 250 150 C 250 300 300 300 300 450 A 50 50 0 1 0 450 450 L 550 300";
 }
+
+/*
+
+```cpp
+auto handle = Hello::create();
+const Hello &instance = *handle;
+assert(!instance.window().is_visible());
+```
+
+
+```rust
+let instance = Hello::new();
+assert!(!instance.window().is_visible());
```

Comment:
i think you could call show() and hide() on the window because it is using the test backend.

Reply:
Good idea. Done.